packages/create-vuestic/src/steps/2.addVuestic/insert-nuxt-module.ts

```diff
@@ -0,0 +1,42 @@
+const createVuesticConfig = (css: string[] = []) => {
+  if (css.length) {
+    return `
+config: {
+  // Vuestic config here
+},
+
+css: [${css.map((c) => `'${c}'`).join(', ')}],
+`.trim()
+  }
+
+  return `
+config: {
+  // Vuestic config here
+}
+`.trim()
+}
+
+const createNuxtConfig = (vuesticConfig: string) => {
+  return `
+  modules: [
+    '@vuestic/nuxt'
+  ],
+
+  vuestic: {
+    ${vuesticConfig.split('\n').join('\n    ')}
+  },
+`
+}
+
+export const insertNuxtModule = (source: string, css: string[]) => {
+  const vuesticConfig = createVuesticConfig(css)
+
+  if (/vuestic: \{/.test(source) || /modules: \[((.|\n)*)\]/gm.test(source)) {
```

Comment:
Does this situation possible? 0_o

Reply:
Not yet. But would be nice if create-vuestic can append vuestic-ui to already existing project in future.